### PR TITLE
description for new system architecture

### PR DESCRIPTION
**Pull request by fanyangCS**

### CODE REVIEW

File: docs/system_architecture.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
-Upon receiving API calls, the REST server coordinates with [FrameworkLauncher](frameworklauncher/README.md) (short for Launcher)
-to perform job management.
-The Launcher Server handles requests from the REST Server and submits jobs to Hadoop YARN.
-The job, scheduled by YARN with [GPU enhancement](https://issues.apache.org/jira/browse/YARN-7481),
-can leverage GPUs in the cluster for deep learning computation. Other type of CPU based AI workloads or traditional big data job
-can also run in the platform, coexisted with those GPU-based jobs.
-The platform leverages HDFS to store data. All jobs are assumed to support HDFS.
-All the static services (blue-lined box) are managed by Kubernetes, while jobs (purple-lined box) are managed by Hadoop YARN.
+<center>
+<img src="./arch_v2.png" width="50%">
```
Comment (squirrelsc) [inline docs/system_architecture.md]: How about put automl another color, or aside? as it's not part of openpai.

File: docs/system_architecture.md [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+
+Job orchestrator and OpenPAI runtime are two key components that understand and execute the workload specified by the OpenPAI protocol. Job orechestrator is implemented by [FrameworkController](https://github.com/Microsoft/frameworkcontroller), a general purpose k8s controller that orchestrates k8s Pods supporting all kinds of AI workloads. The [OpenPAI runtime](../src/kube-runtime) provides runtime support to the workload and implement [OpenPAI runtime parameters/variables](./job_tutorial.md) that are necessary to support the OpenPAI protocol. 
+OpenPAI runtime also prebuilds with [failure analysis](../src/job-exit-spec/config/user-manual.md) rules that can detect typical runtime failure patterns. OpenPAI might take some actions against the detected failure pattens. For example, if OpenPAI finds the job failure is due to Python syntax error, it will not retry the job regardless of the job retry behavior specified by user to prevent from unnecessary retry and the corresponding waste of cluster resource.
```
Comment (yqwang-ms) [inline docs/system_architecture.md]: Job History is also a distinguished feature compared with other platforms, should we include it here?

File: docs/system_architecture.md [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
+The hardware and software requirements include GPU/CPU/Memory resource requirement, docker images, data/code location, the training method (gang scheduling or elastic), job completion policy, etc.
+OpenPAI protocol facilitates platform interoperability and job portability, a job described by the protocol can run on differnt clusters managed by OpenPAI, as long as the clusters can meet the specification. 
+The OpenPAI protocol also enables great flexibility, any AI workload, being it Tensorflow, PyTorch, or your proprietory deep learning workload, can be described by the protocol.
+
+Job orchestrator and OpenPAI runtime are two key components that understand and execute the workload specified by the OpenPAI protocol. Job orechestrator is implemented by [FrameworkController](https://github.com/Microsoft/frameworkcontroller), a general purpose k8s controller that orchestrates k8s Pods supporting all kinds of AI workloads. The [OpenPAI runtime](../src/kube-runtime) provides runtime support to the workload and implement [OpenPAI runtime parameters/variables](./job_tutorial.md) that are necessary to support the OpenPAI protocol.
```
Comment (squirrelsc) [inline docs/system_architecture.md]: It's better to have a bullet list to introduce components one by one.

File: docs/system_architecture.md (hunk 1 above)
Comment (fanyangCS) [reply docs/system_architecture.md]: You are right. I remove automl.

File: docs/system_architecture.md (hunk 3 above)
Comment (fanyangCS) [reply docs/system_architecture.md]: the figure is not a complete list of component. this serves more like a highlight.

File: docs/system_architecture.md (hunk 2 above)
Comment (fanyangCS) [reply docs/system_architecture.md]: you are right. but the description is already too long :-)

**Pull request merged**